src/index.html

```diff
@@ -87,12 +90,40 @@ <h2 class="modal__title">New Post</h2>
         </div>
       </div>
     </div>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button" class="modal__close-button"></button>
+        <h2 class="modal__title">Edit Profile Avatar</h2>
+        <form class="modal__form" id="avatar-form">
+          <label for="avatar-picture" class="modal__label">
+            Profile Picture
+            <input id="avatar-input" type="url" class="modal__input" placeholder="Paste the link to an image" required>
+            <span class="modal__error" id="avatar-input-error">Please enter a URL.</span>
```

Comment:
`Please enter a URL.`


Please, delete the initial error texts from `index.html` because they should appear only when the user is typing anything in the inputs